app/views/users/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="container">
+    <div class="user-box">
+        <img class="image-user" src=<%= @user.photo %> alt="user"/>
+        <div class="user-info">
+            <p class="user-name"><%= @user.name %></p>
+            <p class="user-post">Number of posts: <%= @user.posts_counter%></p> 
+        </div>   
+    </div>   
+    <p class="user-bio"><%= @user.bio %></p>
+    
+    <% @user.recent_posts.each do |post| %>
+        <%= link_to user_post_path(@user.id, post.id) do %>
+        <div class="user-recent-post">
+            <p class="post-title">Post : <%= post.title %></p>
+            <p class="post-text"><%= post.text %></p>
+            <h3 class="counters">Comments: <%= post.comments_counter %>
+                Likes: <%= post.likes_counter %><h3>             
+        </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a default value to the `Likes counter` and ``comments counter`` to make your app more professional. You can update it to something like this:
```ruby
<h3 class="counters">Comments: <%= post.comments_counter || 0 %>Likes: <%= post.likes_counter || 0 %><h3>
```